delete.go

```diff
@@ -188,6 +197,11 @@ func (b *DeleteBuilder) What(what ...string) *DeleteBuilder {
 	return b
 }
 
+func (b *DeleteBuilder) Using(table string) *DeleteBuilder {
```

Comment:
According to [the doc](https://www.postgresql.org/docs/10/static/sql-delete.html), `USING` actually accepts a list of table names. It would be great to implement, should be similar to multi-delete case.